### PR TITLE
fixed typo in lib/index.js

### PR DESCRIPTION
**Pull request by sfabriece**
Fixed a type in internals.validateCallback.

### CODE REVIEW

**Comment by AdriVanHoudt** [comment]
nice, thanks
It looks like you removed a lot of spaces after function and the newline at the end of the file. If you fix those I'll merge.

**Comment by sfabriece** [comment]
Damn editor, i got learn to check that. :)

**Comment by AdriVanHoudt** [comment]
No problem, it's only small stuff, btw you get linting feedback from the tests ;) (although there are some more linting issues but I'll fix those if you don't :p)

**Comment by sfabriece** [comment]
How is it now?

File: lib/index.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ var Hoek = require('hoek');
 
 var internals = {};
 
-internals.validateCallback = function (err, isValid, credentials, reply) {
```
Comment (AdriVanHoudt) [inline lib/index.js]: fix these, as in space after function ;)

**Comment by sfabriece** [comment]
If every styling problem isn't fixed now, I don't know what is. :smile:

**Comment by AdriVanHoudt** [comment]
you need to leave the spaces after function so `function ()` instead of `function()` ;)
and I'll be out of office for a week so I'll merge when I get back, thanks for the effort!

**Pull request closed without merging**